src/radio-button/test/changeEvent.test.ts

```diff
@@ -23,7 +23,8 @@ describe("RadioButton", () => {
       const inputsEl = el.querySelectorAll(
         ".kuc-radio-button__group__select-menu__item__input"
       );
-      inputsEl[2].dispatchEvent(new Event("change"));
+      const focusElm = inputsEl[2] as HTMLInputElement;
+      focusElm.focus();
```

Comment:
How about unifying the variable name with `inputsEl` as below?
```suggestion
      const focusEl = inputsEl[2] as HTMLInputElement;
      focusEl.focus();
```